test/optimizations/sungeun/optimized-on/test_OptimizedOnAtomic.chpl

```diff
@@ -0,0 +1,24 @@
+
+on Locales[numLocales-1] {
+  var y:atomic int;
+
+  on Locales[0] {
+
+    on Locales[numLocales-1] {
+      local atomic_fetch_add_explicit_int_least64_t(y._v, 1, memory_order_seq_cst);
```

Comment:
Ok. I don't really love the idea of explicitly calling runtime functions or using the "private" _v field, but I won't fight too hard on that.